encoding.go

```diff
@@ -44,21 +44,21 @@ func bit(bitlist []byte, nr int) bool {
 var serializedPayloadTooShort = errors.New("verkle payload is too short")
 
 func ParseNode(serialized []byte, depth byte, comm []byte) (VerkleNode, error) {
-	if len(serialized) < 64 {
+	if len(serialized) < 1+StemSize+SerializedAffinePointSize {
```

Comment:
Transforming magic numbers into more obvious constant calculations. This wasn't trivial now that the size of the affine point changed it's size.